pl_examples/hydra_examples/conf/trainer_conf.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass
+from typing import Union, Optional, List, Dict, Tuple, Iterable, Any
+
+import hydra
+from hydra.core.config_store import ConfigStore
+
+
+@dataclass
+class LightninghTrainierConf:
+    # callbacks: Optional[List[Callback]] = None
```

Comment:
This one can't be configured in general as it wants the constructed object. @omry what could be a good way of supporting this?

Reply:
We can handle it later. I have a few ideas.